src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -9443,6 +9647,15 @@ export class StudyViewPageStore
         submitToPage(url, formOps, '_blank');
     }
 
+    onVisualizingMutationPlot(hugoGeneSymbol: string): void {
```

Comment:
This is not a good function name. Should be something like

```suggestion
    onShowMutaitonsInResultsView(hugoGeneSymbol: string): void {
```